Makefile.dist

```diff
@@ -61,4 +59,5 @@ dist:
 	@$(MAKE) -f Makefile ghostunnel.man
 	@mv ghostunnel.man "${CURRENT_DIR}/dist/ghostunnel-${VERSION}.man"
 	@$(MAKE) -f "${MKFILE_PATH}" -j1 build
+	@cd "${CURRENT_DIR}/dist/" && sha256sum ghostunnel-* > sha256sums.txt
```

Comment:
would this actually append instead of override?

Reply:
It should overwrite, since I used `>` (as opposed to `>>`).

Reply:
Also, to be clear, we want to overwrite here. At this point in the script all binaries have been created and we sha256sum them all at once.

Reply:
Good context, yes this is exactly what I was wondering here.